### PR TITLE
adding the nsidc's tutorial

### PR DESCRIPTION
**Pull request by egreckase**

### CODE REVIEW

**Review by scottyhq — CHANGES_REQUESTED** [review]
thanks @egreckase see failing check. Please 'restart notebook and clear all outputs' for `nsidc-data-access.ipynb`

**Comment by egreckase** [comment]
thanks Scott!  I cleared the outputs and restarted the notebook :)

**Comment by scottyhq** [comment]
>  I cleared the outputs and restarted the notebook :)

perfect. now, you'll just have to add the notebook into the table of contents
`tutorials/nsidc-data-access/nsidc-data-access`

this link will bring up the editor on your `nsidc-branch`:
https://github.com/snowex-hackweek/website2022/edit/nsidc-branch/book/_toc.yml

**Comment by egreckase** [comment]
I attempted to add our tutorial to the table of contents, but it appears to be failing the checks and I'm uncertain why.

File: book/_toc.yml [hunk 1]
```diff
@@ -46,6 +46,10 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+         - file: tutorials/nsidc-data-access/index
+           title: Accessing SnowEx Data at the NSIDC DAAC
+           sections: 
+            - file: tutorials/nsidc-data-access/nsidc-data-acces
```
Comment (scottyhq) [inline book/_toc.yml]:
```suggestion
           sections: 
             - file: tutorials/nsidc-data-access/nsidc-data-access
```
make sure file names are correct and match indents with other sections

Comment (egreckase) [reply book/_toc.yml]: thank you!

**Review by scottyhq — APPROVED** [review]
thanks @egreckase ! going to merge this and you can make additional changes if you want in a separate pull request

**Pull request merged**